go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/projectdiscovery/nuclei/v3
 
-go 1.21
+go 1.22.0
```

Comment:
we stay on go 1.21 for now

Reply:
The minimum Go version required for that lib is 1.22. But I'll give it a try to see if it's compatible.